modules/test-renderers.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const nunjucks = require('nunjucks');
+nunjucks.configure('views');
+
+function renderComponentMacro(componentPath, macroName, params) {
```

Comment:
This is slightly experimental but explores writing snapshot tests for individual form macros. It has a big caveat that anything relying on context data outside of explicitly passed props needs to be mocked. As component macros should _largely_ be self-contained this feels like a reasonable trade-off but we'll find out as we add more tests.